includes/Analytics.php

```diff
@@ -71,7 +73,7 @@ public function get_tracking_id(): string {
 		 *
 		 * @var string $tracking_id
 		 */
-		$tracking_id = $this->settings->get_setting( $this->settings::SETTING_NAME_TRACKING_ID );
+		$tracking_id = (string) $this->settings->get_setting( $this->settings::SETTING_NAME_TRACKING_ID );
```

Comment:
Curious to hear why this was added.

It shouldn't be needed because it ought to be a string already.

Reply:
To the lints pass.

Reply:
But where did things start to fail now?

Reply:
https://github.com/GoogleForCreators/web-stories-wp/actions/runs/3412221592/jobs/5677445754#step:9:22

Reply:
That looks more like an issue with PHPStan than anything.

The `@var` block above should mark this as a string and nothing else.

In fact, after rebasing with `main` and removing this cast, there are no PHPStan errors anymore.

So this can be reverted.

Reply:
Reverted. https://github.com/GoogleForCreators/web-stories-wp/pull/12631/commits/80419786946f7a7a52471df74314a0057143f588